Makefile

```
@@ -758,6 +761,117 @@ bench-chromium-request-all: build setup-default
+# block into a running campaign:
+#     corpus_campaign.sh: line 201: not: command not found
+# which killed a three-cell sweep after its first cell had already gated clean.
+#
+# reqbench.sh already solves this FOR ITSELF: it stages its five sources plus
+# the two binaries into a content-addressed bundle, chmod 0555, and execs the
+# copy. This applies the same discipline one layer up, from make, so a campaign
+# is immune to edits in the working tree for its whole lifetime. Everything for
+# one run -- the frozen orchestrator, its manifest, the source revision, and
+# every reqbench record -- lands under a single directory.
+CORPUS_STAMP ?= $(shell date +%Y%m%d-%H%M%S)
+CORPUS_RUN_DIR ?= $(CURDIR)/bench/chromium/results/corpus-$(CORPUS_STAMP)
```

checkmake (0.3.2)

[warning] Variable "CORPUS_STAMP" possibly contains a timestamp and should be simply expanded.

(timestampexpanded)